components/appmgt/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/handlers/proxy/ReverseProxyHandler.java

```diff
@@ -1,6 +1,22 @@
-package org.wso2.carbon.appmgt.gateway.handlers.proxy;
+/*
+*  Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
```

Comment:
Remove Year rage from copyright header, Correct header only contain current year only. This has to be fixed in all new files.
Eg:
Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.